ledger/daml-on-sql/README.rst

```diff
@@ -0,0 +1,742 @@
+assigned to each transaction. This effectively means that having two Ledger API servers
+running on top of a single PostgreSQL server can lead to duplicated and inconsistent
+offsets, effectively leading to undefined (and likely broken) behavior. For this reason,
+it's important to maintain a strict 1:1 relationship between a running Ledger API server
+and a running PostgreSQL server. Note that using PostgreSQL in a high-availability
+configuration does not allow you to run additional Ledger API servers.
+
+Downtime for the Ledger API server can be minimized using a watchdog or orchestration
+system taking care of evaluating its health of the core components and ensuring its
+availability. The Ledger API implementation of DAML-on-SQL exposes the standard gRPC
+health checkpoint that can be used to evaluate the health status of the Ledger API
+component. More information on the endpoint can be found at the documentation for gRPC.
+
+Security and privacy
+********************
+
+Trust assumptions
+=================
+
+In DAML-on-SQL, all data is kept centrally by the operator of the deployment.
+Thus, it's their responsibility to ensure that the data is treated with the
+appropriate care so to respect confidentiality and the applicable regulations.
+
+The ledger operator is advised to use the tools available to them to not divulge
+private user data, including those documented by PostgreSQL, to protect data at
+rest and using a secure communication channel between the Ledger API server and
+the PostgreSQL server.
+
+Ledger API over TLS
+===================
+
+To protect data on-the-wire and allow using the Ledger API over untrusted networks,
+the DAML-on-SQL Ledger API server implementation leverages gRPC's built-in TLS support
+to allow clients to verify the server's identity and encrypt the communication channel
+over which the Ledger API requests and responses are sent.
+
+To enable TLS, you need to specify the private key for your server and the certificate
+chain via ``java -jar daml-on-sql-<version>.jar --pem server.pem --crt server.crt``.
+By default, DAML-on-SQL requires client authentication as well. You can set a custom root
+CA certificate used to validate client certificates via ``--cacrt ca.crt``. You can
+change the client authentication mode via ``--client-auth none`` which will disable it
+completely, ``--client-auth optional`` which makes it optional or specify the default
+explicitly via ``--client-auth require``.
+
+Ledger API Authentication
+=========================
+
+By default, DAML-on-SQL does not use any authentication and accepts all valid Ledger API requests.
+
+To start DAML-on-SQL with authentication based on `JWT <https://jwt.io/>`__ tokens,
+use one of the following command line options:
+
+- ``--auth-jwt-rs256-crt=<filename>``.
+  \DAML-on-SQL will expect all tokens to be signed with RS256 (RSA Signature with SHA-256) with the public key loaded from the given X.509 certificate file.
```

Comment:
What does the `\` at the start of the line do?

Reply:
It's a literary device that I often use to confuse people when copying and pasting text. :smile:

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6850/commits/9b2bfdd6d54ce32d0e2bb17c994cd1afb64ba3bb